kusd/backend.go

```diff
@@ -142,12 +142,15 @@ func New(ctx *node.ServiceContext, config *Config) (*Kowala, error) {
 	kusd.ApiBackend.gpo = gasprice.NewOracle(kusd.ApiBackend, gpoParams)
 
 	// consensus validator
-	networkContract := getNetworkContract(kusd.BlockChain(), NewContractBackend(kusd.ApiBackend))
+	election, err := network.NewElection(NewContractBackend(kusd.ApiBackend), chainConfig.ChainID)
+	if err != nil {
+		log.Crit("Failed to load the network contract", "err", err)
```

Comment:
do we Panic, or should we just not start the Validator?

Reply:
We need to panic as the validator & api depend on this interface - even if the validator is not running you may need to use this interface (ex: redeemDeposits()).

Reply:
That give a new perspective, we can access the Validation network even if we are not a validator or validating. So in future I think we going to end up with 2 diff interfaces